suitcase/utils/__init__.py

```diff
@@ -1,25 +1,219 @@
+from pathlib import Path
```

Comment:
Can you alphabetize the imports? It makes it easier to quickly check if something is already imported or needed to be added.